frontend/src/Commons/Header/Header.tsx

```diff
@@ -0,0 +1,31 @@
+import { useState, useEffect } from 'react';
+import { screenSizes } from '../../Constants/ScreenSize';
+import { StyledHeader, Logo, Title } from './Header.style';
+
+const Header = () => {
+  const [isDesktop, setIsDesktop] = useState<boolean>(false);
+  useEffect(() => {
+    const handleResize = () => {
+      setIsDesktop(window.innerWidth >= parseInt(screenSizes.desktop, 10));
+    };
+
+    // Initial check
+    handleResize();
+
+    // Change Logo based on state
+    window.addEventListener('resize', handleResize);
+
+    // Logo changes and removes what's mounted
+    return () => {
+      window.removeEventListener('resize', handleResize);
+    };
+  }, []);
+  return (
+    <StyledHeader>
+      <Logo src={isDesktop ? '../../../public/assets/BCID-logo-desktop.png' : '../../../public/assets/BCID-logo-mobile.png'} alt="bc-logo" />
```

Comment:
1. Import them at top of the file, it's cleaner
2. I believe images in `public` can have their imports started with just `/` reducing the need of back pedalling with `../..`

Reply:
added imports for images for clarity. For example
import mobileLogo from 'path'